pkg/v1/tkg/test/tkgctl/shared/e2e_common.go

```diff
@@ -150,6 +155,14 @@ func E2ECommonSpec(context context.Context, inputGetter func() E2ECommonSpecInpu
 		By(fmt.Sprintf("Waiting for workload cluster %q nodes to be up and running", clusterName))
 		framework.WaitForNodes(framework.NewClusterProxy(clusterName, tempFilePath, ""), 2)
 
+		By(fmt.Sprintf("Verify addon packages on workload cluster %q matches clusterBootstrap info on management cluster", clusterName))
+		scheme := runtime.NewScheme()
+		err = pkgiv1alpha1.AddToScheme(scheme)
+		Expect(err).NotTo(HaveOccurred())
+		// check antrea package information
+		err = checkPackageInstalls(context, client, scheme, "antrea")
```

Comment:
This e2e_common seems to be called for both legacy (non clusterclass) and clusterclass based clusters. For now lets only call this function if its clusterclass.i.e. if cluster plan == devcc. If not lets skip this. 

Later we need to enhance this function to work for legacy clusters as well